packages/alert-dialog/src/alert-dialog.css

```diff
@@ -0,0 +1,17 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@import url('./spectrum-alert-dialog.css');
+
+:host([variant='scroll']) {
+    max-height: 90vh;
+}
```

Comment:
Is this really a style of the dialog, or is this something that is applied by the context in which an Alert Dialog is delivered?

Reply:
Nope this was coming from spectrum-Modal class as in the image below. Currently we are detached from the modal so to achieve the scroll variant I have added the max-height.
One catch I think is if this alert dialog is added in a modal then we might have two scrolls??

<img width="1364" alt="Screenshot 2023-09-07 at 5 49 04 PM" src="https://github.com/adobe/spectrum-web-components/assets/8790510/9d37350f-7d66-4b60-a715-2f53e430237f">

Reply:
Removed this